### PR TITLE
Fixing Logger not having permissions to create logs

### PR DESCRIPTION
**Pull request by CK-USAF**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -181,24 +181,24 @@ SQL_QUEUE_CONNECTION_STRING=postgresql://postgres:my_password@host.docker.intern
 SQL_QUEUE_NAME=queue_name
 ```
 
-The container can then be spun up using the below command.
+The container can then be spun up using the below command. ```<name>``` in the command is whatever you would like to call the volume where the logs will be stored. The most likely path of where the logs are actually written to on your machine is `/var/lib/docker/volumes/<name>/_data`.
```
Comment (AM-USAF) [inline README.md:184]: I would give it a more descriptive name, like the name of what, log_name, log_folder, volume_name something more than just name

File: Dockerfile [hunk 2]
```diff
@@ -1,7 +1,8 @@
 FROM python:3.11-slim
-WORKDIR /home
-COPY . ./
-RUN pip install .
 RUN useradd -ms /bin/bash default
+WORKDIR /home/default
+COPY --chown=default:default . ./
```
Comment (AM-USAF) [inline Dockerfile:4]: what is this line doing?

Comment (CK-USAF) [reply Dockerfile:4]: which line?

File: start_services.sh [hunk 3]
```diff
@@ -1,13 +1,9 @@
-cd task_queue
-
 if [ $1 == "controller" ]; then
-    cd cli
     echo "Runner controller cli with args ${@:2}"
-    python work_queue_service_cli.py "${@:2}"
+    python task_queue/cli/work_queue_service_cli.py "${@:2}"
```
Comment (JR-USAF) [inline start_services.sh:3]: May as well change this to be python3 as well for consistency with the uvicorn call

**Review by JR-USAF — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -181,24 +181,24 @@ SQL_QUEUE_CONNECTION_STRING=postgresql://postgres:my_password@host.docker.intern
 SQL_QUEUE_NAME=queue_name
 ```
 
-The container can then be spun up using the below command.
+The container can then be spun up using the below command. `<volume_name>` in the command is whatever you would like to call the volume where the logs will be stored. The most likely path of where the logs are actually written to on your machine is `/var/lib/docker/volumes/<volume_name>/_data`.
```
Comment (JR-USAF) [inline README.md:185]: I wasn't able to find this path on my local machine, where do the logs show up if running the image on windows?

File: Dockerfile (hunk 2 above)
Comment (AM-USAF) [reply Dockerfile:4]: Oh sorry, 4. From my understanding, the chown is giving the default group ownership of the whole repo?

**Review by JR-USAF — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (CK-USAF) [reply README.md:185]: use ```docker volume inspect <volume_name>```

File: Dockerfile (hunk 2 above)
Comment (CK-USAF) [reply Dockerfile:4]: yeah its giving default rights over everything in default and everything that might be created in default

**Review by JR-USAF — APPROVED** [review]

File: Dockerfile (hunk 2 above)
Comment (CK-USAF) [reply Dockerfile:4]: that line was added by Ethan

**Review by JR-USAF — APPROVED** [review]

**Review by AM-USAF — APPROVED** [review]

**Pull request merged**